theme/inc/template-tags.php

```diff
@@ -49,11 +49,11 @@ function braces_post_nav() {
 	}
 	?>
 	<nav class="navigation post-navigation" role="navigation">
-		<h1 class="screen-reader-text"><?php _e( 'Post navigation', 'braces' ); ?></h1>
+		<h1 class="screen-reader-text"><?php esc_html_e( 'Post navigation', 'braces' ); ?></h1>
 		<div class="nav-links">
 			<?php
-				previous_post_link( '<div class="nav-previous">%link</div>', _x( '<span class="meta-nav">&laquo;</span> %title', 'Previous post link', 'braces' ) );
-				next_post_link(     '<div class="nav-next">%link</div>',     _x( '%title <span class="meta-nav">&raquo;</span>', 'Next post link',     'braces' ) );
+				previous_post_link( '<div class="nav-previous">%link</div>', esc_html_x( '<span class="meta-nav">&laquo;</span> %title', 'Previous post link', 'braces' ) );
+				next_post_link(     '<div class="nav-next">%link</div>',     esc_html_x( '%title <span class="meta-nav">&raquo;</span>', 'Next post link',     'braces' ) );
 			?>
```

Comment:
Same deal as above: Calling esc_html_x  on lines 55 and 56 will cause HTML tags in the string to be escaped, which I believe is not what you want.